runtime/v2/shim/shim_windows.go

```diff
@@ -79,6 +80,14 @@ func handleSignals(logger *logrus.Entry, signals chan os.Signal) error {
 	}
 }
 
+func openLog() (io.Writer, error) {
+	l, err := winio.ListenPipe("log", nil)
```

Comment:
Same here. This leaks the listener right?

Reply:
It would also be more useful I think to name it like: [util_windows.go](https://github.com/containerd/containerd/blob/master/runtime/v2/shim/util_windows.go#L47) so it can be predicted?

```go
fmt.Sprintf("\\\\.\\pipe\\containerd-shim-%s-%s-log-pipe", ns, id)
```